docs/setup/install-new.md

```diff
@@ -174,8 +174,9 @@ Kubernetes. We will leverage it to install the `application-gateway-kubernetes-i
 Now that we have App Gateway, AKS, and AGIC installed we can install a sample app
 via [Azure Cloud Shell](https://shell.azure.com/):
 
- ```bash
```

Comment:
can you revert this ?

Reply:
Did it not fix the rendering? If you look at the picture none of that is being rendered as a code block

Reply:
But I can revert it no problem if need be

Reply:
as in this part, `cat <<EOF | kubectl apply -f -` and `EOF` part.
Formatting change above looks good.

Reply:
done 😄